packages/job-worker/src/blueprints/context/adlibActions.ts

```diff
@@ -64,27 +65,22 @@ export enum ActionPartChange {
 	SAFE_CHANGE = 1,
 }
 
-/** Actions */
-export class ActionExecutionContext extends ShowStyleUserContext implements IActionExecutionContext, IEventContext {
-	private readonly _context: JobContext
-	private readonly _cache: CacheForPlayout
-	private readonly rundown: DBRundown
-	private readonly playlistActivationId: RundownPlaylistActivationId
+export class DatastoreActionExecutionContext
+	extends ShowStyleUserContext
+	implements IDataStoreActionExecutionContext, IEventContext
+{
+	protected readonly _context: JobContext
 
 	/** To be set by any mutation methods on this context. Indicates to core how extensive the changes are to the current partInstance */
 	public currentPartState: ActionPartChange = ActionPartChange.NONE
```

Comment:
These currentPartState and nextPartState properties look unused, could they be removed?